themes/V3/5ePHB/snippets.js

```diff
@@ -21,9 +21,26 @@ module.exports = [
 		view      : 'text',
 		snippets  : [
 			{
-				name : 'Table of Contents',
-				icon : 'fas fa-book',
-				gen  : TableOfContentsGen
+				name        : 'Table of Contents',
```

Comment:
we might want to change those icons to something like `h1-h4`

Reply:
I was thinking in the lines of this:
<img width="670" alt="image" src="https://github.com/naturalcrit/homebrewery/assets/91662974/4801430f-da7a-4736-91e4-9c906b7b39bb">

But those are pro icons, and we don't have access to that. I guess it is fine as it is.